Gems/Atom/RPI/Code/Source/RPI.Builders/Material/MaterialTypeBuilder.cpp

```diff
@@ -493,7 +493,8 @@ namespace AZ
                 }
 
                 // Intermediate azsl file
-                AZStd::string generatedAzsl = AZStd::string::format("// This code was generated by %s. Do not modify.\n\n", MaterialTypeBuilderName);
+                AZStd::string generatedAzsl;
+                generatedAzsl += AZStd::string::format("// This code was generated by %s. Do not modify.\n", MaterialTypeBuilderName);
```

Comment:
Why was this moved from being initialized in the constructor?

Reply:
There were different versions of the file when I merged. One version have the lines split like this further down in the file. That's the version I kept. History of changes was lost when I retargeted to development branch.